### DISCUSSION TITLE
mac m1 arm support

### DISCUSSION THREAD

**Discussion by Stek0v** [OGSR/OGSR-Engine#438] [Q&A] [answered]
Ребят, а можно будет собрать под arm (m1)?

**Comment by xrSimpAI** [comment, marked as answer]
Нет, OGSR только для Windows. Тебе надо в @OpenXRay - там собирают Xray на Linux, ARM и тп.

**Comment by Xottab-DUTY** [comment]
Собственно, да, у нас как раз под M1 и собирается. Правда, со свежим SDK macOS какие-то проблемы, но на старой версии собирается и работает :)
